.editorconfig

```diff
@@ -0,0 +1,17 @@
+root = true
+
+[{configure,{*.{c,cpp,h,go,java,js,py,rs}}]
```

Comment:
is it worth including toml here?

Reply:
If you think it's worth it, I'd put it under its own section like

```
[*.toml]
charset = utf-8
end_of_line = lf
insert_final_newline = true
trim_trailing_whitespace = true
```

As AFAICT it's just a flat INI style file with no indentation...